.gitattributes

```diff
@@ -0,0 +1,17 @@
+# Improve diff output for PHP files.
+*.php              diff=php
```

Comment:
TIL! Oh wow, we need this on `amp-wp`.

Where is this documented?

Reply:
TIL as well. It's documented here: https://www.git-scm.com/docs/gitattributes#_defining_a_custom_hunk_header

Reply:
Will the endless deluge of tips and tricks ever end?